controllers/restic.go

```diff
@@ -184,7 +184,7 @@ func (r *DPAReconciler) buildResticDaemonset(dpa *oadpv1alpha1.DataProtectionApp
 	// Update Items in ObjectMeta
 	dsName := ds.Name
 	ds.TypeMeta = installDs.TypeMeta
-	ds.Labels, _ = common.AppendUniqueLabels(ds.Labels, installDs.Labels)
+	ds.Labels, _ = common.AppendUniqueKeyTOfTMaps(ds.Labels, installDs.Labels)
```

Comment:
Why are we skipping err processing here ?

Reply:
+1